src/components/summit-event-bulk-actions/summit-event-bulk-editor-item.js

```diff
@@ -84,6 +84,14 @@ class SummitEventBulkEditorItem extends React.Component
         value = value.valueOf()/1000;
         let { event, currentSummit } = this.props;
         let eventModel = new SummitEvent(event, currentSummit);
+        if(event.end_date) {
```

Comment:
review https://github.com/fntechgit/summit-admin/blob/ae519789d108172c81fe7ab2dbb294862914342e/src/components/forms/event-form.js#L204
lets use the same logic
and refactor so on both places we use the same logic
knowledge should exist only on one place